src/ontology/uberon-edit.obo

```diff
@@ -53380,7 +53379,7 @@ property_value: homology_notes "On the internal or parietal surface of the left
 [Term]
 id: UBERON:0002495
 name: long bone
-def: "Long bone is a limb bone that is subcylindrical and has a shaft with periosteum separating the ends of the bones. Long bones are present only in the limbs[VSAO:wd]." [https://github.com/obophenotype/uberon/issues/32, VSAO:wd, Wikipedia:Long_bone]
+def: "A long bone is a bone that has a shaft and 2 ends and is longer than it is wide. Long bones have a thick outside layer of compact bone and an inner medullary cavity containing bone marrow. The ends of a long bone contain spongy bone and an epiphyseal line. The epiphyseal line is a remnant of an area that contained hyaline cartilage that grew during childhood to lengthen the bone." [https://github.com/obophenotype/uberon/issues/2010, https://github.com/obophenotype/uberon/issues/32, Wikipedia:Long_bone] {comment="https://medlineplus.gov/ency/imagepages/9582.htm"}
 comment: Endochondral_ossification is an essential process during the rudimentary formation of long bones, with the exception of the clavicle[Wikipedia:Endochondral_ossification]. The medial and and lateral ends undergo EO, the mid-portion is formed by a process with features of EO & IO (the process is shared by the mandible)[ISBN:9780397517251]
```

Comment:
In the latest commit I have
- switched an `annotationProperty` from `comment` to `database_cross_reference`,
- changed the definition of `long bone` so that it works across species and across stages,
- changed the ticket references so that they use `seeAlso`.